cime_config/machines/config_machines.xml

```diff
@@ -3073,6 +3073,130 @@
     </resource_limits>
   </machine>
 
+  <machine MACH="aurora">    
+    <DESC>ANL Aurora ECP machine, Aurora consists of multiple racks, each with many nodes, for a total of over 10,000 nodes, batch system is pbspro</DESC>
+    <NODENAME_REGEX>aurora-uan-.*</NODENAME_REGEX>    <OS>LINUX</OS>
+    <COMPILERS>oneapi-ifx,oneapi-ifxgpu,oneapi-ifort,gnu</COMPILERS>
+    <MPILIBS>mpich,impi,openmpi</MPILIBS>
+    <CHARGE_ACCOUNT>CSC249ADSE15_CNDA</CHARGE_ACCOUNT>
+    <SAVE_TIMING_DIR>/lus/gecko/CSC249ADSE15_CNDA/performance_archive</SAVE_TIMING_DIR>
+    <SAVE_TIMING_DIR_PROJECTS>.*</SAVE_TIMING_DIR_PROJECTS>
+    <CIME_OUTPUT_ROOT>/lus/gecko/projects/CSC249ADSE15_CNDA/$USER/scratch</CIME_OUTPUT_ROOT>
+    <DIN_LOC_ROOT>/lus/gecko/projects/CSC249ADSE15_CNDA/inputdata</DIN_LOC_ROOT>
```

Comment:
What project is "CSC249ADSE15_CNDA"

Reply:
This is for ECP Aurora early access project, it is valid through Apr. 2024.